draft-decraeneginsberg-lsr-isis-fast-flooding.xml

```diff
@@ -202,136 +203,145 @@
       achieving a consistent LSDB on all nodes in the network could easily
       take a minute or more.</t>
 
-      <t>Increasing the LSP flooding rate therefore becomes an essential element
+			<t>Increasing the LSP flooding rate therefore becomes an essential element
       of supporting greater network scale.</t>
 
-      <t>
+			<t>
 	Improving the LSP flooding rate is complementary to protocol
 	extensions that reduce LSP flooding traffic by reducing the
 	flooding topology such as Mesh Groups <xref target="RFC2973"/>
-	or Dynamic Flooding <xref
-	target="I-D.ietf-lsr-dynamic-flooding"/>. Reduction of the
+	or Dynamic Flooding <xref target="I-D.ietf-lsr-dynamic-flooding"/>
+. Reduction of the
```

Comment:
This re-filling of the paragraph introduces an incorrect space after the reference.

Reply:
OK. Thank you